src/shapes/polyline.class.ts

```diff
@@ -134,6 +142,16 @@
       };
     },
 
+    /**
+     * After setting scale, recalculates dimensions when the stroke is uniform and stroke line join is bevel or round
+     * @private
+     */
+    _set(key, value) {
+      var output = this.callSuper('_set', key, value);
```

Comment:
`output` equals `this` currently
But maybe it will change.
Good enough, let's leave as is.